designer/client/page-edit.js

```diff
@@ -25,10 +26,19 @@ export class PageEdit extends React.Component {
       section: page?.section ?? "",
       isEditingSection: false,
       errors: {},
+      toggle: false,
     };
     this.formEditSection = React.createRef();
   }
 
+  async componentDidMount() {
+    const result = await new ToggleApi().fetchToggles();
+
+    this.setState({
+      toggle: result["featureEditPageDuplicateButton"] == true,
```

Comment:
what is `result["featureEditPageDuplicateButton"]`? A boolean? do you need to do the equality check? also, `===`?

If result["featureEditPageDuplicateButton"] is a boolean (or undefined) you can do this:

```suggestion
    const result = await new ToggleApi().fetchToggles();
    const { featureEditPageDuplicateButton ?? true, otherFeature ?? false } = result //when destructuring, if featureEditPageDuplicateButton is nullish, default to true

    this.setState({
      featureEditPageDuplicateButton,
      otherFeature
      
```